package.json

```diff
@@ -26,7 +26,7 @@
   "homepage": "https://github.com/UKHomeOffice/asl#readme",
   "dependencies": {
     "@asl/components": "^3.9.3",
-    "@asl/pages": "^15.18.0",
+    "@asl/pages": "^15.18.1",
```

Comment:
I don't actually care that much, as everything is still in development, but this was a pretty huge change so it should at least have been a minor release, I may have even made it a major as I doubt it is backwards compatible

Reply:
generally only use patch for small tweaks and bugfixes

Reply:
done